COLLADAStreamWriter/src/COLLADASWNode.cpp

```diff
@@ -17,9 +17,9 @@ namespace COLLADASW
 {
 
     //---------------------------------------------------------------
-    void Node::start()
+    void Node::start(bool forceNonInstancedNode)
     {
-        if ( !mIsInstanceNode )
+		if (!mIsInstanceNode || forceNonInstancedNode)
```

Comment:
forceNonInstancedNode is a way to force to export node as an instanceNode.
With nodes in LodGroup, we want them to be exported not in the visualScene as children of lodGroupNode. We want them to be exported in libraryNode.

For example for LOD1 in the visual scene, we have : 
 <node id="_lodGroup1_pCube1_LOD1" name="pCube1_LOD1" type="NODE">

But with LOd Group we want:

<node id="LOD___lodGroup1_pCube1_LOD1" name="_lodGroup1_pCube1_LOD1" type="NODE">
      <instance_node url="#_lodGroup1_pCube1_LOD1">
 ...

so _lodGroup1_pCube1_LOD1 was Node and now is an instance Node